manifests/olm-catalog/4.7/nfd.v4.7.0.clusterserviceversion.yaml

```diff
@@ -34,7 +34,7 @@ metadata:
     olm.skipRange: ">=4.1.0 <4.7.0"
```

Comment:
@ArangoGutierrez skipRange needs an update, otherwise we will never update to 4.7

Reply:
Done